data/base-skills.yaml

```diff
@@ -0,0 +1,112 @@
+---
+
+skillsets:
+  Armor:
+    - Shield Usage
+    - Light Armor
+    - Chain Armor
+    - Brigandine
+    - Plate Armor
+    - Defending
+    # Guild Skills
+    - Conviction # Paladin
+  Weapon:
+    - Parry Ability
+    - Small Edged
+    - Large Edged
+    - Twohanded Edged
+    - Small Blunt
+    - Large Blunt
+    - Twohanded Blunt
+    - Slings
+    - Bow
+    - Crossbow
+    - Staves
+    - Polearms
+    - Light Thrown
+    - Heavy Thrown
+    - Brawling
+    - Offhand Weapon
+    - Melee Mastery
+    - Missile Mastery
+    # Guild Skills
+    - Expertise # Barbarian
+  Magic:
+    - Primary Magic # See `guild_skill_aliases` section
+    - Arcana
+    - Attunement
+    - Augmentation
+    - Debilitation
+    - Targeted Magic
+    - Utility
+    - Warding
+    - Sorcery
+    # Guild Skills
+    - Astrology # Moon Mage
+    - Summoning # Warrior Mage
+    - Theurgy # Cleric
+    - Inner Magic # Thief
+    - Inner Fire # Barbarian
+    - Lunar Magic # Moon Mage, Trader
+    - Elemental Magic # Warrior Mage, Bard
+    - Holy Magic # Cleric, Paladin
+    - Life Magic # Empath, Ranger
+    - Arcane Magic # Necromancer
+  Survival:
+    - Evasion
+    - Athletics
+    - Perception
+    - Stealth
+    - Locksmithing
+    - Thievery
+    - First Aid
+    - Outdoorsmanship
+    - Skinning
+    # Guild Skills
+    - Scouting # Ranger
+    - Backstab # Thief
+    - Thanatology # Necromancer
+  Lore:
+    - Alchemy
+    - Appraisal
+    - Enchanting
+    - Engineering
+    - Forging
+    - Outfitting
+    - Performance
+    - Scholarship
+    - Tactics
+    # Guild Skills
+    - Empathy # Empath
+    - Bardic Lore # Bard
+    - Trading # Trader
+    # Deprecated Skills
+    - Mechanical Lore
+
+# Because the Primary Magic skill is labeled differently per guild...
+# In game, you'd type `exp primary magic` to see the rank, but the XML data
+# sent and parsed by Lich is the guild-specific label. Therefore, this cross-reference
+# helps scripts to determine the correct value to use with DRSkills.getxp('Skill Name').
+guild_skill_aliases:
```

Comment:
Primary Magic is the only aliased skill that I'm aware of at the moment. I figured if it happened once, it could happen again, and this approach would be a relatively simple config change to add in new skill aliases.